charts/rancher-windows-gmsa/3.0.0/app-readme.md

```diff
@@ -1,7 +1,12 @@
 # Windows GMSA Admission Webhook
 
-This chart creates the GMSA CRD, Credential, and Admission Webhook. The official documentation and tutorials can be found [here](https://github.com/kubernetes-sigs/windows-gmsa).
-
+This chart deploys the [Windows GMSA Admission Webhook](https://github.com/kubernetes-sigs/windows-gmsa/tree/master) onto a Windows cluster.
+This admission webhook allows workloads in the cluster to specify `securityContext.windowsOptions.gmsaCredentialSpecName` to target an existing `GMSACredentialSpec` CRD that the workload's `ServiceAccount` is permitted to `use`.
+If this requirement is met, the webhook will populate `securityContext.windowsOptions.gmsaCredentialSpec` with the contents of the `GMSACredentialSpec` resource.
+The official documentation and tutorials can be found [here](https://github.com/kubernetes-sigs/windows-gmsa).
+> **Note**: If `credential.enabled` is set to `true, a default `GMSACredentialSpec` that can be used by workloads will also be created.
+>
+> However, workloads will need to introduce their own RBAC that gives their `ServiceAccount` permissions to `use` this `GMSACredentialSpec`.
```

Comment:
Remove all of this since it already exists in the `README.md`, so it will be displayed on the Rancher page.